package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/metarhia/filestorage#readme",
   "scripts": {
-    "test": "eslint . && metatests test; rm -r root-directory",
+    "test": "eslint . && metatests test; rm -r test/fs-root",
```

Comment:
`rm -r test/fs-root` this should be done in the tests where this directory is created, not globally.

Reply:
I don't think `metatests` provide an option to do something after all tests run e.g. cleanup and since all tests are async there is no way we know which test will run last. Removing dir after each test is not an option because they share the same root directory and having separate dir for each test doesn't look good to me.

Reply:
You can listen on the `'done'` event of the parent test to know when all of its subtests are done.

You can just use an empty parent:
```js
const fsTest = metatests.test('Filestorage write tests');
fsTest.test('1', test => ...);
fsTest.test('2', test => ...);
fsTest.test('3', test => ...);

fsTest.endAfterSubtests();
fsTest.on('done', () => cleanup());
```

Also, it's not a bad idea to have separate directories for each test as this will ensure that they don't interfere with each other and test exactly what they are supposed to without side effects. tbh I'd prefer this way.